editoast/src/views/speed_limit_tags.rs

```diff
@@ -0,0 +1,24 @@
+use actix_web::get;
+use actix_web::web::Json;
+
+use crate::error::Result;
+use crate::generated_data::speed_limit_tags_config::SpeedLimitTag;
+
+crate::routes! {
+    "/speed_limit_tags" => {
+        speed_limit_tags,
+    },
+}
+
+#[utoipa::path(
+    tag = "speed_limit_tags",
+    responses(
+        (status = 200, description = "List of configured speed-limit tags", body = Vec<String>, example = json!(["V200", "MA80"])),
+    ),
+)]
+#[get("")]
+async fn speed_limit_tags() -> Result<Json<Vec<String>>> {
+    let speed_limit_tags_config = SpeedLimitTag::load();
```

Comment:
Good idea, done in https://github.com/OpenRailAssociation/osrd/pull/8106/commits/e379f7c4c2816f42cddc3c79360b2d20e330beab

Note that it could be extended to sprites (`/signaling_systems` endpoint), but I didn't dig that.
If you think I should, please shout 🙂